website/templates/include/profile/social.mako

```diff
@@ -106,7 +106,7 @@
                 <label>Academia</label>
                 <div class="input-group">
                 <span class="input-group-addon">https://</span>
-                <input class="form-control" data-bind="value: academiaInstitution" placeholder="institution"/>
+                <input class="form-control" data-bind="value: academiaInstitution" placeholder="institution" size="5"/>
```

Comment:
Regarding @sloria's [comment on 4782](https://github.com/CenterForOpenScience/osf.io/pull/4782#issuecomment-169375462) that this change is supposed to resolve, it may be best to have someone product/frontend-related review the relative sizing. Setting size to `1` seems about as large as necessary:

<img width="661" alt="screen shot 2016-01-14 at 15 44 34" src="https://cloud.githubusercontent.com/assets/5659262/12337455/374e7404-bad8-11e5-9349-ba2cce48505e.png">
Compared to `5`:
<img width="659" alt="screen shot 2016-01-14 at 15 44 53" src="https://cloud.githubusercontent.com/assets/5659262/12337460/40ffc21e-bad8-11e5-9d2e-6bc079af3d16.png">